two-entity-poc/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "two-entity-poc",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.ts",
+  "scripts": {
+    "start": "ts-node-dev src/app.ts",
+    "dev": "nodemon src/app.ts",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "sidharth.jain@zemosolabs.com",
+  "license": "ISC",
+  "devDependencies": {
+    "@types/express": "^4.17.21",
+    "@types/node": "^20.12.8",
+    "nodemon": "^3.1.0",
+    "ts-node-dev": "^2.0.0"
+  },
+  "dependencies": {
+    "@types/sequelize": "^4.28.20",
+    "body-parser": "^1.20.2",
+    "dotenv": "^16.4.5",
+    "express": "^4.19.2",
+    "express-validator": "^7.0.1",
+    "mysql2": "^3.9.7",
+    "sequelize": "^6.37.3"
```

Comment:
in your next node project.

use sequelize-typescript for creating sequelize models

Reply:
okay.